package.json

```diff
@@ -10,17 +10,26 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.11",
+    "@react-navigation/bottom-tabs": "^6.3.2",
+    "@react-navigation/material-bottom-tabs": "^6.2.2",
     "@react-navigation/native": "^6.0.11",
     "@react-navigation/native-stack": "^6.7.0",
     "@react-navigation/stack": "^6.2.2",
     "expo": "~45.0.0",
     "expo-status-bar": "~1.3.0",
+    "global": "^4.4.0",
```

Comment:
This package is not necessary, remove it. Please do not add unnecessary dependencies without consulting the team first.

Reply:
Removed this unnecessary package